.rubocop.yml

```diff
@@ -49,6 +49,12 @@ RSpec/DescribeClass:
 RSpec/ExampleLength:
   Enabled: false
 
+# FIXME: Find a better way to block output during specs, and fix relevant examples.
+RSpec/ExpectOutput:
```

Comment:
👍 Feel free to ping me whenever you're doing rubocop-rspec stuff. I get a lot out of seeing how people interact with the cops.